install_config/storage_examples/shared_storage.adoc

```diff
@@ -13,4 +13,462 @@ toc::[]
 == Overview
 The following use case describes how a user wanting to leverage shared storage for use by two seperate containers would configure the solution. This example highlights the use of NFS, but can easily be adapted to other shared storage types, such as GlusterFS. In addition, this example will show configuration of pod security as it relates to shared storage.
 
+For an explanation of persistent volumes (PVs), persistent volume claims (PVCs), and
+using NFS as persistent storage, please see the section on
+link:../persistent_storage/persistent_storage_nfs[Persistent Storage Using NFS].
+Below, is an end-to-end example of using an existing nfs cluster as an OpenShift persistent store.
+It is assumed that a working NFS server and exports  has already been setup. 
+
+
+[NOTE]
+====
+All `oc ...` commands are executed on the OpenShift master node.
+====
+
+=== 1. Create the Persistent Volume
```

Comment:
Goes for this and the rest of the section titles: I would take the numbers out and move these sections after Overview up a level (i.e., `==`). Also use the gerund form for titles, when possible (e.g., "Creating the Persistent Volume").